spec/jobs/hirmeos_work_updater_job_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Hyrax::Hirmeos::HirmeosWorkUpdaterJob do
+  let(:work) { create(:work_with_one_file) }
+
+  describe '#perform' do
+    let(:service) { instance_double(Hyrax::Hirmeos::MetricsTracker, submit_diff_to_hirmeos: true) }
+
+    before do
+      allow_any_instance_of(described_class).to receive(:service).and_return(service)
```

Comment:
Do you think it would be better to stub the client instead of the tracker?
```suggestion
      allow(Hyrax::Hirmeos::Client).to receive(:initialize).and_return(client)
```
then you could do something like:
```
expect(client).to have_received(:post_files).once
```